website/project/model.py

```diff
@@ -397,10 +397,19 @@ class NodeLog(StoredObject):
     REGISTRATION_APPROVAL_INITIATED = 'registration_initiated'
     REGISTRATION_APPROVAL_APPROVED = 'registration_approved'
 
+    @classmethod
+    def actions(cls):
```

Comment:
Talked to Sam. Not really. It's meant to be immutable, and that may have been what he was thinking. He does not have strong feelings on the subject.